lib/contestant.rb

```diff
@@ -0,0 +1,23 @@
+class Contestant
+  attr_reader :first_name, :last_name, :full_name, :age, :state_of_residence, :game_interests
```

Comment:
No need for an `attr_reader` for first_name or last_name here, because we only ever need to call `.full_name`.